tests/particles/test_particle_class.py

```diff
@@ -1573,3 +1573,27 @@ def test_undefined_mass_energy() -> None:
     """Test that a particle with an undefined mass energy returns |nan| J."""
     nu_tau_particle = Particle("nu_tau")
     assert u.isclose(nu_tau_particle.mass_energy, np.nan * u.J, equal_nan=True)
+
+
+@pytest.mark.parametrize(
+    ("particle_symbol", "expected_ionization_energy"),
+    [
+        ("H", u.Quantity((13.598434599702 * u.eV).to(u.J).value, u.J)),
```

Comment:
I think this is equivalent, and a little clearer?
```suggestion
        ("H", (13.598434599702 * u.eV).to(u.J) ),
```